monacaのCordova 11環境で、Androidアプリ設定にてターゲットSDKバージョンを31指定すると、ビルドエラーになります

## 不具合の概要(Description of bug)

- SDKのバージョン(SDK version): NIFCloudMB（ncmb-push-monaca-plugin）3.1.1
- 不具合が発生した環境(Environment requirement)
monacaでCordova 11でAndroidアプリ設定にて、ターゲットSDKバージョンを31指定
- 不具合を再現させる手順・サンプルコード(Step for Reproduce bug / Sample code)
   - monacaでCordova 11でAndroidアプリ設定にて、ターゲットSDKバージョンを31指定し、Androidアプリをビルドすると以下のようにエラーが発生しています。
   - Cordova 11は正式に対応しておりませんが、今後の対応にあたって、本現象を改善するように改修を進めています。
- エラーメッセージ(Error messages)
```
[16:31:32] FAILURE: Build failed with an exception.
[16:31:32] * What went wrong:
[16:31:32] Execution failed for task ‘:app:processDebugMainManifest’.
[16:31:32] > Manifest merger failed : Apps targeting Android 12 and higher are required to specify an explicit value for `android:exported` when the corresponding component has an intent filter defined. See https://developer.android.com/guide/topics/manifest/activity-element#exported for details.
[16:31:32] * Try:
[16:31:32] Run with
```

--- a/plugin.xml
+++ b/plugin.xml
@@ -50,7 +50,8 @@
         </config-file>
         <config-file target="AndroidManifest.xml" parent="/manifest/application">
             <service
-                android:name="com.nifcloud.mbaas.core.NCMBFirebaseMessagingService">
+                android:name="com.nifcloud.mbaas.core.NCMBFirebaseMessagingService"
+                android:exported="false">
                 <intent-filter>
                     <action android:name="com.google.firebase.MESSAGING_EVENT"/>
                 </intent-filter>

